testing/test_config.py

```diff
@@ -341,6 +341,31 @@ def test_invalid_options_show_extra_information(testdir):
         "*  rootdir: %s*" % testdir.tmpdir,
     ])
 
+
+@pytest.mark.parametrize('args', [
+    ['dir1', 'dir2', '-v'],
+    ['dir1', '-v', 'dir2'],
```

Comment:
This case and the next would fail this test before the patch, with `rootdir` being set to `myroot/dir1` and `myroot/dir2` respectively (effectively ignoring the 3rd argument).